src/Data/List/Base.agda

```diff
@@ -395,6 +396,23 @@ deduplicateᵇ : (A → A → Bool) → List A → List A
 deduplicateᵇ r [] = []
 deduplicateᵇ r (x ∷ xs) = x ∷ filterᵇ (not ∘ r x) (deduplicateᵇ r xs)
 
```

Comment:
I think these functions below deserve an opening documentation comment to the effect that the first two are about the *first* occurrence in the list, while the third is about *all* occurrences, rather than having to go back to this originating PR... 

... alternatively, and against existing practice in FP libraries elsewhere, they could instead be named
* `findFirst`
* `findFirstIndex`
* `findAllIndices`

But you (and others here) may find those names a bit too verbose... ;-)?

Reply:
Hmm, this comment (intended to be attached to your revised module L398) now seems to govern the original version... hope this is visible!

Reply:
Sorry to nitpick! The style guide asks for sentence capitalisation of comments, rather than all-lower-case. Can you fix this please?

Reply:
Missed that. Thanks!